res/translations/es/messages.ftl

```diff
@@ -0,0 +1,211 @@
+welcome_title = Welcome
+welcome_message =
+    Space Acres es una aplicación gráfica (GUI) diseñada para unirte como granjero en la Red de Autonomys.
+
+    Antes de continuar, necesitas 3 cosas:
+    ✔ Una cartera donde recibirás las recompensas (utiliza Subwallet, la extensión de polkadot{”{”}.js{”}”} o cualquier otra cartera compatible con la cadena Substrate)
+    ✔ 100 GB de espacio en un SSD de buena calidad para almacenar los datos del nodo
+    ✔ Cualquier SSD (o varios) con tanto espacio como puedas permitirte para fines de agricultura, esto es lo que generará las recompensas
+
+welcome_button_continue = Continuar
+
+upgrade_title = Actualización
+upgrade_message =
+    ¡Gracias por elegir Space Acres de nuevo!
+
+    La blockchain que estabas utilizando antes de la actualización ya no es compatible con esta versión de Space Acres, probablemente porque estabas participando en la versión anterior de Autonomys.
+
+    Pero no te preocupes, puedes actualizarte a la red actual en un solo clic!
+upgrade_button_upgrade = Actualizar a {$chain_name}
+
+loading_title = Cargando
+loading_configuration_title = Cargando configuración
+loading_configuration_step_loading = Cargando configuración...
+loading_configuration_step_reading = Leyendo configuración...
+loading_configuration_step_configuration_exists = Leyendo configuración...
+loading_configuration_step_configuration_not_found = Leyendo configuración...
```

Comment:
These should change despite them being the same string in English. See the key for hints on what it should be.